docs/using-the-compiler.rst

```diff
@@ -323,6 +323,8 @@ Input Description
         },
         // Metadata settings (optional)
         "metadata": {
+          // The metadata is appended at the end of the bytecode by default
```

Comment:
```suggestion
          // The metadata is appended at the end of the bytecode by default.
          // Setting this to false omits the metadata from both runtime and deploy time code.
```

Reply:
Fixed this to "Setting this to false omits the metadata from the deployed code." which feels more accurate.

Reply:
It should remove it from both, not just deployed code.

Reply:
It also removes it from the nested bytecode of any contract created with `new`. Not sure if that's worth mentioning here or not but that's the detail people often forget.

Reply:
What about this? I think it would be a good idea to mention it.

Reply:
I feel that mentioning the nested part is redundant. As an interface, if there is an option to disable metadata, it should disable it everywhere. I can also add a note somewhere in the docs if you want to.

Reply:
Well, we could add a note on the page about metadata.

But I guess it's not that important. If you think it's not a good idea to put it here, let's just drop the idea and have the PR merged :)